### PR TITLE
Young Serif: Version 3.002; ttfautohint (v1.8.4.7-5d5b);gftools[0.9.33.dev8+g029e19f] added

### PR DESCRIPTION
**Pull request by emmamarichal**
e276c34: [gftools-packager] Young Serif: Version 3.002; ttfautohint (v1.8.4.7-5d5b);gftools[0.9.33.dev8+g029e19f] added

* Young Serif Version 3.002; ttfautohint (v1.8.4.7-5d5b);gftools[0.9.33.dev8+g029e19f] taken from the upstream repo https://github.com/noirblancrouge/YoungSerif at commit https://github.com/noirblancrouge/YoungSerif/commit/fd4e8eff6a9fbe0a1368a1a059f5d3d743473309.

 da2d89a: Description added

### CODE REVIEW

**Comment by RosaWagner** [comment]
@emmamarichal 
🔥 FAIL: METADATA.pb: Check URL on copyright string is the same as in repository_url field. ([com.google.fonts/check/metadata/consistent_repo_urls](https://font-bakery.readthedocs.io/en/stable/fontbakery/profiles/googlefonts.html#com.google.fonts/check/metadata/consistent_repo_urls))
🔥 FAIL Value of repository_url field is https://github.com/noirblancrouge/YoungSerif
But font copyright string has: https://github.com//noirblancrouge
[code: mismatch]

The copyright string in font info needs to be corrected.

**Comment by emmamarichal** [comment]
Updated

## Young Serif: Version 3.002; ttfautohint (v1.8.4.7-5d5b);gftools[0.9.33] added

---
 303dadc: [gftools-packager] Young Serif: Version 3.002; ttfautohint (v1.8.4.7-5d5b);gftools[0.9.33] added

* Young Serif Version 3.002; ttfautohint (v1.8.4.7-5d5b);gftools[0.9.33] taken from the upstream repo https://github.com/noirblancrouge/YoungSerif at commit https://github.com/noirblancrouge/YoungSerif/commit/08359934f95dcdcd36d8afee364ced8ba9ba8e68.

 9dead3e: Description added

File: ofl/youngserif/METADATA.pb [hunk 1]
```diff
@@ -0,0 +1,22 @@
...
+  copyright: "Copyright 2023 The Young Serif Project Authors (https://github.com/noirblancrouge/YoungSerif)"
+}
+subsets: "latin"
+subsets: "latin-ext"
+subsets: "menu"
+source {
+  repository_url: "https://github.com/noirblancrouge/YoungSerif"
+  commit: "08359934f95dcdcd36d8afee364ced8ba9ba8e68"
+}
+primary_script: "Zinh"
```
Comment (RosaWagner) [inline ofl/youngserif/METADATA.pb:22]: this primary script is a bit weird

**Comment by RosaWagner** [comment]
Problem of design of oslash and Ldot

Screenshot 2023-07-28 at 13 59 05
Screenshot 2023-07-28 at 13 59 43
Screenshot 2023-07-28 at 14 02 31

Kerning issues

Screenshot 2023-07-28 at 14 03 19
Screenshot 2023-07-28 at 14 02 48
Screenshot 2023-07-28 at 14 02 40

**Comment by emmamarichal** [comment]
Updated

## Young Serif: Version 3.003; ttfautohint (v1.8.4.7-5d5b);gftools[0.9.33] added

---
 447b809: [gftools-packager] Young Serif: Version 3.003; ttfautohint (v1.8.4.7-5d5b);gftools[0.9.33] added

* Young Serif Version 3.003; ttfautohint (v1.8.4.7-5d5b);gftools[0.9.33] taken from the upstream repo https://github.com/noirblancrouge/YoungSerif at commit https://github.com/noirblancrouge/YoungSerif/commit/8d67638d5ae1aa9e339d53326cd608e186d03dbd.

 7254945: description and metadata


 84e7ac4: metadatas

**Comment by emmamarichal** [comment]
@RosaWagner updated! :)

**Pull request merged**